[2025-09-06T05:33:45.988+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:33:42.931526+00:00 [queued]>
[2025-09-06T05:33:46.008+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:33:42.931526+00:00 [queued]>
[2025-09-06T05:33:46.009+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-09-06T05:33:46.039+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): validate_model> on 2025-09-06 05:33:42.931526+00:00
[2025-09-06T05:33:46.063+0000] {standard_task_runner.py:57} INFO - Started process 1782 to run task
[2025-09-06T05:33:46.076+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mysql_insert_select', 'validate_model', 'manual__2025-09-06T05:33:42.931526+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/mysql_insert_select.py', '--cfg-path', '/tmp/tmpjno_u10h']
[2025-09-06T05:33:46.079+0000] {standard_task_runner.py:85} INFO - Job 306: Subtask validate_model
[2025-09-06T05:33:46.178+0000] {task_command.py:410} INFO - Running <TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:33:42.931526+00:00 [running]> on host 6faa580ba6cb
[2025-09-06T05:33:46.348+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_insert_select' AIRFLOW_CTX_TASK_ID='validate_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T05:33:42.931526+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T05:33:42.931526+00:00'
[2025-09-06T05:33:46.350+0000] {logging_mixin.py:149} INFO - ✅ Modelo cargado correctamente
[2025-09-06T05:33:46.350+0000] {logging_mixin.py:149} INFO -    - Tipo: <class 'numpy.ndarray'>
[2025-09-06T05:33:46.350+0000] {logging_mixin.py:149} INFO -    - Características esperadas: No disponible
[2025-09-06T05:33:46.351+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-09-06T05:33:46.368+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=mysql_insert_select, task_id=validate_model, execution_date=20250906T053342, start_date=20250906T053345, end_date=20250906T053346
[2025-09-06T05:33:46.446+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-09-06T05:33:46.506+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
